compute/instances/troubleshooting/cant-connect-to-instance-with-pn-gateway.mdx

```diff
@@ -0,0 +1,25 @@
+---
+meta:
+  title: I cannot connect to my Instance using SSH after after attaching it to a Private Network
+  description: This page explains how troubleshoot connection problems after after attaching an Instance to a Private Network
+content:
+  h1: I cannot connect to my Instance using SSH after after attaching it to a Private Network
+  paragraph: This page explains how troubleshoot connection problems after after attaching an Instance to a Private Network
+tags: troubleshoot error private-network private network vpc dhcp
+dates:
+  validation: 2024-05-24
+  posted: 2021-05-26
+categories:
+  - compute
+---
+
+<Macro id="iam-requirements" />
+
+The action to take depends on whether DHCP is [activated](/network/vpc/reference-content/vpc-migration) on your Private Network.
+
+If it is not, disconnect the Instance from the Private Network, as there may be other factors impacting your Instance, like one of your Instances running a DHCP server.
+
+If it is, this is expected behavior as all the traffic towards your Instance now goes through the Public Gateway. To access your Instance using SSH, first create a static NAT association between a port of your Public Gateway (eg 2222) and the private IP assigned to your Instance, on the SSH port (22 by default). Then, SSH to the Public Gateway's IP on port 2222.
+
+
+alors, ça risque d’arriver, donc peut-être à adapter pour expliquer pourquoi. mais oui une PGW sans route par défaut, c’est pas hyper utile maintenant qu’on enlève DHCP et DNS
```

Comment:
Just marking this comment so it doesn't go unnoticed when publishing 🤓

Reply:
```suggestion
```